### PR TITLE
Added pilot buttons that switch to a panel that displays pilot profil…

### PR DESCRIPTION
**Pull request by maxwellpark**
…es. Uses a prefab & scriptable objects for pilot data. Buttons are created at runtime (similar to mission buttons).

### CODE REVIEW

File: Assets/Scripts/PilotButton.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PilotButton : MonoBehaviour
+{
+    //public GameObject buttonObject;
```
Comment (ambid17) [inline Assets/Scripts/PilotButton.cs]: why is all of this commented out

File: Assets/Scripts/PilotsManager.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+{
+	public GameObject crewPanel;
+	public GameObject pilotButtonPrefab;
+	public GameObject pilotProfilePanel;
+	public PilotsContainer pilotsContainer;
+
+	private void Awake()
+	{
+		pilotProfilePanel.SetActive(false); 
+		pilotProfilePanel.GetComponentInChildren<Button>().onClick.AddListener(delegate { ClosePilotProfilePanel(); });
```
Comment (ambid17) [inline Assets/Scripts/PilotsManager.cs]: just hold a reference to this button, in case the UI changes

Comment (ambid17) [inline Assets/Scripts/PilotsManager.cs]: remember to RemoveAllListeners() first

File: Assets/Prefabs/PilotButton.prefab [hunk 3]
```diff
@@ -0,0 +1,201 @@
+%YAML 1.1
```
Comment (ambid17) [inline Assets/Prefabs/PilotButton.prefab:1]: you need to set the anchors so they scale

**Review by ambid17 — CHANGES_REQUESTED** [review]
Once we get designs from Fardeen, you will need to make the `PilotsProfilePanel` responsive rather than laying the elements just out on the page. However for a first pass it works well for now :D

File: Assets/Scripts/PilotButton.cs (hunk 1 above)
Comment (maxwellpark) [reply Assets/Scripts/PilotButton.cs]: It was redundant. Removed it.

**Comment by maxwellpark** [comment]
Considering making a game-wide UI generation utils class, hence why the PilotsUtils class is unfinished.

**Pull request merged**